app/Libraries/OsuWiki.php

```diff
@@ -40,12 +45,19 @@ public static function getTree(?string $sha = null, bool $recursive = true): arr
         return GitHub::gitData()->trees()->show(static::user(), static::repository(), $sha ?? static::branch(), $recursive);
     }
 
-    public static function fetch($path)
+    public static function fetch($path): array
```

Comment:
I notice the fetch doesn't seem to be used outside so maybe it's fine as private? 🤔 (and type `$path` as string?)

Reply:
Might as well make it `private`, only case I see for it public public is being able to call it directly in console, but we can just the results from `->data` anyway